capabilities/vcs/user-repos/profile.supr

```diff
@@ -45,4 +45,16 @@ usecase UserRepos safe {
     """
     description string!
   }
+
+example success {
```

Comment:
Do you also want to add an example for errors?

Reply:
Added it, but not sure the maps really work like that